### PR TITLE
Update seventeentrack config

### PR DESCRIPTION
**Pull request by McSwindler**
## Proposed change

Updating the seventeentrack docs to conform with the new V1 API configuration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/70261
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #65399

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @McSwindler, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22428).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Review by rjmidau — CHANGES_REQUESTED** [review]
* Update documentation to reflect that the Integration now uses Config Flow instead of YAML
* Explicitly mention that you must create a Developer-type account, not a Buyer or Seller account (which get Permission Denied errors when trying to access api.17track.net). It isn't possible to convert an existing account between types.
* It's not possible to get to the Settings page directly from the URL mentioned in this document. You must click on 'Go to dashboard', then the Settings page (https://api.17track.net/en/admin/settings) is linked off the Dashboard (https://api.17track.net/en/admin/dashboard).

File: source/_integrations/seventeentrack.markdown [hunk 1]
```diff
@@ -15,38 +15,14 @@ The `seventeentrack` sensor platform allows users to get package data tied to th
...
-  required: false
-  type: boolean
-  default: false
-show_delivered:
-  description: Whether sensors should be created for delivered packages.
-  required: false
-  type: boolean
-  default: false
-{% endconfiguration %}
+1. Register for an account: https://user.17track.net/en/register
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown]: Please don't use bare URLs.

File: source/_integrations/seventeentrack.markdown [hunk 2]
```diff
@@ -15,38 +15,14 @@ The `seventeentrack` sensor platform allows users to get package data tied to th
...
+2. Enter the verification code sent to your email.
+3. Select "Developer Account"
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown]:
```suggestion
3. Select "Developer Account".
```

File: source/_integrations/seventeentrack.markdown [hunk 3]
```diff
@@ -15,38 +15,14 @@ The `seventeentrack` sensor platform allows users to get package data tied to th
...
+4. Complete registration, phone number and skype are not required.
+6. Open the settings page: https://api.17track.net/en/admin/settings
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown]: Please don't use bare URLs.

File: source/_integrations/seventeentrack.markdown [hunk 4]
```diff
@@ -15,38 +15,14 @@ The `seventeentrack` sensor platform allows users to get package data tied to th
...
+7. API Token will be displayed under the Security tab
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown]:
```suggestion
7. API Token will be displayed under the Security tab.
```

**Comment by McSwindler** [comment]
@rjmidau Hopefully I have addressed all of your comments :)

File: source/_integrations/seventeentrack.markdown [hunk 5]
```diff
@@ -15,38 +15,14 @@ The `seventeentrack` sensor platform allows users to get package data tied to th
...
+
+{% include integrations/config_flow.md %}
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown:26]: Did you forget to update the frontmatter to indicate config flow support?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/seventeentrack.markdown [hunk 6]
```diff
@@ -5,52 +5,25 @@ ha_category:
...
 ha_iot_class: Cloud Polling
+ha_config_flow: true
 ha_domain: seventeentrack
 ha_platforms:
   - sensor
 ha_integration_type: integration
 ---
 
-The `seventeentrack` sensor platform allows users to get package data tied to their [17track.net](https://www.17track.net/en) account. The platform creates both summary sensors, which show the number of packages in a current state (e.g., "In Transit"), as well as individual sensors for each package within the account.
+The `seventeentrack` sensor platform allows users to get package data tied to their [17track.net](https://api.17track.net/en) account. The platform creates both summary sensors, which show the number of packages in a current state (e.g., "In Transit"), as well as individual sensors for each package within the account.
```
Comment (frenck) [inline source/_integrations/seventeentrack.markdown]: Just noticed this one, as this is an integration now :)

```suggestion
The 17TRACK integration allows users to get package data tied to their [17track.net](https://api.17track.net/en) account. The platform creates both summary sensors, which show the number of packages in a current state (e.g., "In Transit"), as well as individual sensors for each package within the account.
```

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**